### PR TITLE
add code snippets for retrieve_delivery_date function

### PR DESCRIPTION
**Pull request by jchen293**
Add code snippets for retrieve_delivery_date function and fix some typos in other places

### CODE REVIEW

**Comment by jchen293** [comment]
We will need to release Go lib first and then bump the version in this PR. Drafting this PR until we release the feature on CLs

File: official/docs/node/current/carrier-metadata/retrieve.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const EasyPostBetaClient = require('@easypost/api/src/beta/easypost');
+const EasyPostBetaClient = require('@easypost/api/src/easypost');
```
Comment (Justintime50) [inline official/docs/node/current/carrier-metadata/retrieve.js:1]: issue: this needs to be `'@easypost/api'`, can you ensure the other beta examples for Node are corrected to this too?

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: official/docs/node/current/carrier-metadata/retrieve.js (hunk 1 above)
Comment (jchen293) [reply official/docs/node/current/carrier-metadata/retrieve.js]: This is the only place, fixed

**Review by nwithan8 — APPROVED** [review]

**Review by Justintime50 — APPROVED** [review]
Nice work!

**Pull request merged**